pkg/provider/aws.go

```diff
@@ -60,34 +66,73 @@ var (
 	}
 )
 
-var awsSurvey = []*survey.Question{
-	{
-		Name:     "cluster",
-		Prompt:   &survey.Input{Message: "Enter the name of your cluster:"},
-		Validate: validCluster,
-	},
+var initAwsSurvey = []*survey.Question{
 	{
 		Name:     "region",
 		Prompt:   &survey.Select{Message: "What region will you deploy to?", Default: "us-east-2", Options: awsRegions},
 		Validate: survey.Required,
 	},
 }
 
+var defaultAwsSurvey = []*survey.Question{
+	{
+		Name:     "cluster",
+		Prompt:   &survey.Input{Message: "Enter the name of your cluster:"},
+		Validate: validClusterName(AWS),
+	},
+}
+
 func mkAWS(conf config.Config) (provider *AWSProvider, err error) {
 	provider = &AWSProvider{}
-	if err = survey.Ask(awsSurvey, provider); err != nil {
+	if err = survey.Ask(initAwsSurvey, provider); err != nil {
+		return
+	}
+
+	var createCluster bool
+	if err = survey.AskOne(&survey.Confirm{Message: "Do you want to create a new cluster?", Default: true}, &createCluster); err != nil {
```

Comment:
We should also add to this note that if they don't create a cluster, they should only chose a newly created, fresh cluster

Reply:
We can add the note, but I'm not sure how strictly necessary this is since clashing applications can be disabled. I didn't add it in the artifact (yet), but I think the only one left is cert-manager. External DNS is limited to the subdomain set in the workspace so might not clash, but potentially could so we can add a way to disable that as well.

Reply:
I'm thinking more if you have the zalando postgres operator or nginx ingress controller already installed